### PR TITLE
Upload Assignment 9,10,11,12,13,14

### PR DESCRIPTION
**Pull request by Prashant698**

### CODE REVIEW

**Review by sourabh-kanwade — APPROVED** [review]

File: prashant/JavaScript/Assignment12/assignment12.2.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            <button id="one">Create Object one</button>
+            <button id="two">Create Object two</button>
+            <button id="init">Show initial data</button>
+            <button id="getData">Get Data</button>
+            <button id="cal">Calculate</button>
+            <button id="disp">Display</button>
+        </div>
+        <script src="assign12.2.js"></script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment12/assignment12.2.html]: add new line

File: prashant/JavaScript/Assignment13/assignment13.1.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment 13.1</title>
+    </head>
+    <body>
+        <script>
+            let d = new Date("Feb 21, 2012, 3:12");
+            alert(d);
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment13/assignment13.1.html]: add new line

File: prashant/JavaScript/Assignment13/assignment13.2.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                    break;
+                case 6:
+                    alert("SA");
+                    break;
+                default:
+                    break;
+            }
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment13/assignment13.2.html]: add new line

File: prashant/JavaScript/Assignment13/assignment13.3.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <script>
+            let d = new Date(prompt("Enter a date"));
+            function getDateAgo(date, days) {
+                let temp = date.getTime() - 86400000 * days;
+                return new Date(temp);
+            }
+            alert(getDateAgo(d, 3));
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment13/assignment13.3.html]: add new line

File: prashant/JavaScript/Assignment13/assignment13.4.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        <script>
+            let date = new Date(prompt("Enter Date of birth."));
+            function ageCalculator(dob) {
+                let temp = new Date(Date.now() - dob.getTime());
+                return Math.abs(temp.getUTCFullYear() - 1970);
+            }
+            alert(ageCalculator(date));
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment13/assignment13.4.html]: add new line

File: prashant/JavaScript/Assignment14/assignment14.1.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+                    document.write("<br>Inside Marks---Marks " + this.Marks);
+                }
+            }
+            m2 = new Marks(2, "ss", 45);
+            m2.marks();
+            m2.display();
+        </script>
+    </body>
+</html>
+
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment14/assignment14.1.html]: remove extra line

File: prashant/JavaScript/Assignment14/assignment14.2.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+            acc1.display();
+            let acc2 = new BankDet();
+            acc2.getData(855, "$ilkMoney", 5000);
+            acc2.deposit(1000);
+            acc2.withDraw(7800);
+            acc2.withDraw(5500);
+            acc2.display();
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment14/assignment14.2.html]: add new line

File: prashant/JavaScript/Assignment9/assignment9.4.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+            </div>
+        </div>
+        <script
+            src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+            crossorigin="anonymous"
+        ></script>
+    </body>
+</html>
+
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment9/assignment9.4.html]: remove extra lines

File: prashant/JavaScript/Assignment12/assign12.2.js [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+document.getElementById("disp").onclick = () => {
+    result.innerHTML = emp.display();
+};
+document.getElementById("getData").onclick = () => {
+    let name = prompt("Enter a Name");
+    let addr = prompt("Enter a Address");
+    let city = prompt("Enter a city");
+    let salary = parseInt(prompt("Enter a salary"));
+    emp.get_data(name, addr, city, salary);
+};
```
Comment (Pawan1005) [inline prashant/JavaScript/Assignment12/assign12.2.js]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**